pkg/intermediate/aggregate.go

```diff
@@ -606,8 +606,9 @@ func validateDataRecord(record entities.Record) bool {
 	return true
 }
 
-// isCorrelationRequired returns true when flowType is InterNode and
-// egressNetworkPolicyRuleAction is not deny (drop/reject).
+// isCorrelationRequired returns true when flowType is InterNode,
+// egressNetworkPolicyRuleAction is not deny (drop/reject) and
```

Comment:
When I see the logic below.. it looks like either egressNetworkPolicyRuleAction is not deny or ingressNetworkPolicyRuleAction is not reject. Please use "either.. or.."  in the sentence and not "and".